src/AD.FsCheck.MSTest/AD.FsCheck.MSTest.csproj

```diff
@@ -1,10 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
+    <TargetFramework/>
```

Comment:
Not a fan of this, but it's necessary to unset the TargetFramework property set in Directory.Build.props.

Reply:
Yes, it's not pretty. But for now I think it's ok.